src/ILVerify/src/AccessVerificationHelpers.cs

```diff
@@ -262,15 +262,19 @@ private static bool GrantsFriendAccessTo(this ModuleDesc module, ModuleDesc frie
 
             foreach (var attribute in assembly.GetDecodedCustomAttributes("System.Runtime.CompilerServices", "InternalsVisibleToAttribute"))
             {
-                var friendValues = ((string)attribute.FixedArguments[0].Value).Split(new string[] { ", " }, StringSplitOptions.None);
+                var friendValues = ((string)attribute.FixedArguments[0].Value).Split(',');
                 if (friendValues.Length >= 1 && friendValues.Length <= 2)
                 {
-                    if (friendValues[0] != friendName.Name)
```

Comment:
You may rather use `AssemblyName(string)` constructor to parse this. The full assembly name parser has number of corner cases (escaping, ...). Not worth to replicate it here.